wiki/608037-PHIAnnotationDeIdentification.md

```diff
@@ -42,6 +42,11 @@ Benchmarking:
 
 - Submission queue: `NLP sandbox - Date Annotator`
 - Submission quota: 2 submissions / team / week
+- CPU: 1
```

Comment:
Can you add the frequency, e.g. `1 core @ 2.2 MHz`

Reply:
@tschaffter , theres not actually a core limit though.  The true core limit is the number of cores on the instance.

Reply:
@thomasyu888 Can we limit the number of cores? What is the frequency of the cores?

Reply:
@tschaffter , This is the same issue that we ran into for the EHR challenges.  Since theres no easy way of tracking the CPU's that are in use, and because we run all submissions on one instances, there is currently not a way to limit the number of cores.

Reply:
Thanks @thomasyu888. We need to come up with a new deployment method. Can you please add `CPU: x @ y GHz` or `CPU: x cores of an AWS Graviton Processor` where `AWS Graviton Processor` may be replaced by the suitable proc name. I found this example [here](https://aws.amazon.com/ec2/instance-types/).

Reply:
@thomasyu888 In case you've missed the above comment.